var/spack/repos/builtin/packages/compadre/package.py

```diff
@@ -37,6 +38,12 @@ class Compadre(CMakePackage):
     variant("mpi", default=False, description="Enable MPI support")
     depends_on("mpi", when="+mpi")
 
+    patch(
```

Comment:
Can you add a brief comment above the directive explaining what the patch is for?

Reply:
Just added the comment on patch purpose. Thanks!